<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/prefix1/icon.ico</normaloff>:/new/prefix1/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>510</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="playerLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>player</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>开始(&amp;S)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_S_2">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="actionSdg"/>
   </widget>
   <addaction name="menu_S"/>
   <addaction name="menu_S_2"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/filenew.png</normaloff>:/new/prefix1/image/filenew.png</iconset>
   </property>
   <property name="text">
    <string>新游戏(&amp;N)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/exit.png</normaloff>:/new/prefix1/image/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/gtk-help.png</normaloff>:/new/prefix1/image/gtk-help.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="actionSdg">
   <property name="icon">
    <iconset>
     <normaloff>image/help-about.png</normaloff>image/help-about.png</iconset>
   </property>
   <property name="text">
    <string>开/关音效</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
